tasks/styles.js

```diff
@@ -49,7 +49,9 @@ module.exports = (gulp, $, pkg) => {
       .pipe($.cssUrlCustomHash({
         targetFileType: ['jpe?g', 'png', 'webp', 'svg', 'gif', 'ico', 'otf', 'ttf', 'eot', 'woff2?'],
       }))
-      .pipe($.autoprefixer())
+      .pipe($.autoprefixer({
+        grid: true
```

Comment:
@wrux Is `true` the correct value? From the [documentation](https://github.com/postcss/autoprefixer#options), it looks like it needs to be `autoplace`.

Reply:
Interesting. It looks like using `true` is deprecated now:

> "no-autoplace": enable Autoprefixer grid translations but exclude autoplacement support. You can also use /* autoprefixer grid: no-autoplace */ in your CSS. (alias for the deprecated true value)

Reply:
@colourgarden I think we can just enable prefixing in SASS on a per-project basis and close this PR because grid is not used in most projects. I assume grid autoprefixing grid is disabled by default so the task does not take too long to execute.

Reply:
Will you add an option to enable `grid` prefixing then? With the default set to false?

Also, if the project doesn't use `grid` then autoprefixer won't produce anything extra even if the option is enabled 😉 

So - as long as there are no side effects for non-`grid` projects - maybe it's worth just adding it as a default to futureproof the module.